src/apps/content-editor/src/app/views/ItemEdit/components/ItemEditHeader/ItemEditHeaderActions.tsx

```diff
@@ -135,6 +150,54 @@ export const ItemEditHeaderActions = ({
       { skip: !itemZUID || !modelZUID }
     );
 
+  useEffect(() => {
+    if (
+      isLoadingVersions ||
+      isFetchingVersions ||
+      isFetching ||
+      !isCheckingUrlPathChange
+    )
+      return;
```

Comment:
I suggest formatting this properly, it kind of looks confusing at a glance. Maybe make sure it's at least on the same line above or add an empty line after the return